model/OrganisationStatusV2.ts

```diff
@@ -0,0 +1,13 @@
+import { isly } from "isly"
+import { OrganisationStatus } from "./OrganisationStatus"
+
+export type OrganisationStatusV2 = typeof OrganisationStatusV2.values[number]
+
+export namespace OrganisationStatusV2 {
+	export const values = ["active", "inactive", "deleted"] as const
+	export const type = isly.string<OrganisationStatusV2>(values)
+	export const is = type.is
+	export function toUpperCase(value: OrganisationStatusV2): OrganisationStatus {
+		return value.toUpperCase() as OrganisationStatus
+	}
```

Comment:
to upper case isnt a very good name as it doesnt really tell you whats happening, just how - toLegacy or something? to OrganisationStatusV1?

also is it not possible to have it as a member function, rather than static like this?